robocop/checkers/naming.py

```diff
@@ -262,7 +262,7 @@
         msg="'{{ statement_name }}' is deprecated since Robot Framework version "
         "{{ version }}, use '{{ alternative }}' instead",
         severity=RuleSeverity.WARNING,
-        version=">=4.0",  # TODO change to 5.0 (and tests too)
+        version=">=5.0",
```

Comment:
Switched it back to 4.0 - in previous PR we decided that deprecated-statement can support 4.0 (but only for Run Keyword Unless)